src/lib/utils/synapseTypes/ExternalFileHandleInterface.ts

```diff
@@ -22,14 +21,10 @@ export type ExternalObjectStoreFileHandle = ExternalFileHandleInterface & {
 
 export function assertIsExternalFileHandle(
   x?: FileHandle,
-): asserts x is ExternalFileHandle {
-  if (
-    x?.concreteType !== ExternalFileHandleConcreteTypeEnum.ExternalFileHandle
-  ) {
-    throw new assert.AssertionError({
-      message: ` ExternalFileHandle expected but found ${x}`,
-    })
-  }
+): x is ExternalFileHandle {
+  return (
+    x?.concreteType === ExternalFileHandleConcreteTypeEnum.ExternalFileHandle
+  )
```

Comment:
Now that this returns a boolean rather than an Error being thrown, maybe rename to simply isExternalFileHandle?